src/pupil_labs/neon_recording/stream/av_stream/base_av_stream.py

```diff
@@ -69,16 +69,22 @@ def __init__(
         )
         parts_ts = []
         video_readers = []
-        for av_file, time_file in av_files:
+        av_initial_pts = 0
+        for file_index, (av_file, time_file) in enumerate(av_files):
             if self.kind == "video":
                 part_ts = Array(time_file, dtype=TIMESTAMP_DTYPE)  # type: ignore
                 container_timestamps = (part_ts["ts"] - recording.start_ts) / 1e9
                 reader = plv.Reader(av_file, self.kind, container_timestamps)
                 part_ts = part_ts[: len(reader)]
             elif self.kind == "audio":
+                # scene is not guaranteed to be loaded before audio
```

Comment:
Is this necessary? If scene isn't loaded, calling `rec.scene.ts[0]` would load it.